internal/server/server.go

```diff
@@ -167,6 +167,8 @@ func (s *Server) ListenAndServe(addr string) error {
 	mux.HandleFunc("/v1/auth/accounts/", s.HandleAccounts)
 	mux.HandleFunc("/v2/auth/snapshot", s.HandleSnapshotV2)
 	mux.HandleFunc("/v2/auth/snapshot/", s.HandleSnapshotV2)
+	mux.HandleFunc("/v2/auth/validate-snapshot", s.HandleValidateSnapshotV2)
```

Comment:
Maybe just `/validate` instead of `validate-snapshot`?
```suggestion
	mux.HandleFunc("/v2/auth/validate", s.HandleValidateSnapshotV2)
```